submissions/ik-web/html-css-popup/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML-CSS-Popup</title>
+    <link rel="stylesheet" href="css/app.css">
+</head>
+<body>
+
+    <!-- Nav container -->
+    <nav class="nav">
+        <ul class="nav-inner">
+            <li><a href="#" class="nav-item nav-link">Mail</a></li>
+            <li><a href="#" class="nav-item nav-link">Images</a></li>
+
+            <!-- Popup menu container -->
+
+            <!-- Nav popup button -->
+            <li>
+                <input type="checkbox" id="popup">
+                <label class="nav-item nav-icon" for="popup" tab><img src="icons/popup-button.png" alt="popup-button"></label>
```

Comment:
Please, hide your `input` checkbox
![Screenshot_110](https://user-images.githubusercontent.com/66283273/99445423-30411600-2926-11eb-8daf-7d4c48a36bb6.png)

Reply:
Oh, I tested it in chrome and everything was fine there. But I haven't tested it in other browsers ... I'm sorry ... I changed this in the code and tested it in Mozilla too. It's not easy to be a beginner))

Reply:
No problem, it's okay :+1: